[BUG] Fix LNBits installation bug + misc. improvements

VajraOfIndra opened the pull request:
#### What

The main goal of this PR is to fix a bug in the installation of LNBits. The guide is currently missing the following command: `./venv/bin/python build.py` as is shown in the [LNBits installation guidelines](https://github.com/lnbits/lnbits/blob/main/docs/guide/installation.md#option-3-venv).

I also removed the installation command `./venv/bin/pip install pylightning` because it is not mentionned in the instructions.. is it still needed?

This PR adds this comamnd after editing the configuration file.

This PR contains also the following minor edits:
- Replaces the LNBits data folder name from `/data/LNBits` to `/data/lnbits` -> to follow naming convention so far (only small letters)
- Switching from creating a config file from scratch to copying and editing the existing example file -> it allows for all the configuration options to be present for the user in case they want to tweak it.
- Added a selection of the colour theme with an illustration of the what the colours are -> because the names of the themes are meaningless
- Switched from NGINX to nginx -> for uniformity between guides
- Removed `Wants=lnd.service` from systemd service file and added `PartOf=lnd.service` (see explanations below in 3rd messages)
- Added a "LNBits in action" section -> a short section to explain the first steps of creating a wallet and pointing to useful resources for the user to learn more about the extensions and LNBit wallets
- Updated the Upgrade section to find the latest version

### Why

The guide is NOT working as it is at the moment.

#### Scope

- [ ] significant change to core configuration
- [ ] independent bonus guide
- [x] simple bug fix

#### Test & maintenance

To be tested with a new installation of LNBits from scratch.

twofaktor commented on guide/bonus/lightning/lnbits.md:
```diff
@@ -113,38 +113,90 @@ Table of contents
   $ python3 -m venv venv
```
Line 111-112 why this? official guide installation says this:
```
git clone https://github.com/lnbits/lnbits-legend.git
cd lnbits-legend/
```
https://github.com/lnbits/lnbits/blob/main/docs/guide/installation.md#option-3-venv

VajraOfIndra replied:
I think it was originally for ease of use, i.e. a shorter GitHub local repo name. But happy to use the official (if very confusing) name. Fixed.

blckbx replied:
they recently renamed the repo: `https://github.com/lnbits/lnbits.git` so we can just clone without renaming.

twofaktor commented on guide/bonus/lightning/lnbits.md:
````diff
@@ -113,38 +113,90 @@ Table of contents
+* Copy the example configuration file and open it.
 
   ```sh
+  $ cp .env.example .env
````
Why this? official guide installation says this:
`mkdir data && cp .env.example .env`

VajraOfIndra replied:
The second part of the command is identical.

The first part of the command (making the data directory) is made earlier in the guide when we create the `/data/lnbits` dirctory and its symlink in the "lnbits" user home folder.

The default LNBits installation guidelines create the data directory within the GitHub local repository. For us it would be at: `/home/lnbits/lnbits-legend/data` Instead we have it at `/data/lnbits` and have a symlink to it.

twofaktor reviewed (changes requested):
some suggestions!

VajraOfIndra commented:
I forgot to mention that I added the `PartOf=lnd.service` option in the systemd unit file.

This option will automatically shut down LNBits if LND is shut down. But if LNBits is stopped, LND is NOT stopped. And if LNBits is started but LND is not, it does NOT ask LND to start (to avoid unwanted start of LND), contrary to `Wants=` which I removed for this reason.

twofaktor commented:
> I forgot to mention that I added the `PartOf=lnd.service` option in the systemd unit file.
> 
> This option will automatically shut down LNBits if LND is shut down. But if LNBits is stopped, LND is NOT stopped. And if LNBits is started but LND is not, it does NOT ask LND to start (to avoid unwanted start of LND), contrary to `Wants=` which I removed for this reason.

Amazing very helpful! Could we apply this parameter in BTC RPC Explorer, Fulcrum/Electrs, and other services that depend on the other services? eg. BTC RPC Explorer depends on Bitcoind + Electrs/Fulcrum, Fulcrum/Electrs depend on bitcoind, etc...

VajraOfIndra commented:
> Amazing very useful! Could we apply this parameter too in BTC RPC Explorer, Fulcrum/Electrs, and other services that depend on the other services? eg. BTC RPC Explorer depends on Bitcoind + Electrs/Fulcrum, Fulcrum/Electrs depend on bitcoind, etc...

Yes, I think it'd make more sense. It would also fix the BTC RPC Explorer mishandling of the cookie file, see [comment here in #848](https://github.com/raspibolt/raspibolt/issues/848#issuecomment-1256046700).

blckbx commented on guide/bonus/lightning/lnbits.md:
```diff
@@ -93,69 +93,121 @@ Table of contents
+  
+  # LndRestWallet
+  LND_REST_ENDPOINT=https://127.0.0.1:8080/
+  LND_REST_CERT="/home/bob/.config/Zap/lnd/bitcoin/mainnet/wallet-1/data/chain/bi>
+  LND_REST_MACAROON="/home/bob/.config/Zap/lnd/bitcoin/mainnet/wallet-1/data/chai>
```
paths seems cut off

twofaktor replied:
```
LND_REST_CERT="/home/bob/.config/Zap/lnd/bitcoin/mainnet/wallet-1/data/chain/bitcoin/mainnet/tls.cert"
LND_REST_MACAROON="/home/bob/.config/Zap/lnd/bitcoin/mainnet/wallet-1/data/chain/bitcoin/mainnet/admin.macaroon or HEXSTRING"
```

VajraOfIndra replied:
Fixed

blckbx commented on guide/bonus/lightning/lnbits.md:
````diff
@@ -93,69 +93,121 @@ Table of contents
 
   ```sh
-  $ chmod 600 /home/lnbits/lnbits/.env
+  $ chmod 600 /home/lnbits/lnbits-legend/.env
````
path adjustment 
```suggestion
  $ chmod 600 /home/lnbits/lnbits/.env
```

blckbx commented on guide/bonus/lightning/lnbits.md:
````diff
@@ -93,69 +93,121 @@ Table of contents
 
   ```sh
-  $ cd ~/lnbits
+  $ cd ~/lnbits-legend
````
```suggestion
  $ cd ~/lnbits
```

blckbx commented on guide/bonus/lightning/lnbits.md:
```diff
@@ -179,14 +231,14 @@ Your browser will display a warning because we use a self-signed SSL certificate
 
   [Service]
-  WorkingDirectory=/home/lnbits/lnbits
+  WorkingDirectory=/home/lnbits/lnbits-legend
```
```suggestion
  WorkingDirectory=/home/lnbits/lnbits
```

blckbx commented on guide/bonus/lightning/lnbits.md:
```diff
@@ -179,14 +231,14 @@ Your browser will display a warning because we use a self-signed SSL certificate
+  WorkingDirectory=/home/lnbits/lnbits-legend
 
-  ExecStart=/bin/sh -c 'cd /home/lnbits/lnbits && ./venv/bin/uvicorn lnbits.__main__:app --port 5000'
+  ExecStart=/bin/sh -c 'cd /home/lnbits/lnbits-legend && ./venv/bin/uvicorn lnbits.__main__:app --port 5000'
```
```suggestion
  ExecStart=/home/lnbits/lnbits/venv/bin/uvicorn lnbits.__main__:app --port 5000
```

blckbx commented on guide/bonus/lightning/lnbits.md:
````diff
@@ -257,12 +326,14 @@ Updating to a [new release](https://github.com/lnbits/lnbits-legend/releases){:t
 
   ```sh
-  $ cd /home/lnbits/lnbits
+  $ cd /home/lnbits/lnbits-legend
````
```suggestion
  $ cd /home/lnbits/lnbits
```

blckbx commented on guide/bonus/lightning/lnbits.md:
````diff
@@ -93,69 +93,121 @@ Table of contents
   ```sh
-  $ git clone --branch 0.9.4 https://github.com/lnbits/lnbits-legend lnbits
-  $ cd lnbits
+  $ git clone --branch 0.9.4 https://github.com/lnbits/lnbits-legend
+  $ cd lnbits-legend
````
```suggestion
  $ git clone --branch 0.9.4 https://github.com/lnbits/lnbits
  $ cd lnbits
```

blckbx commented on guide/bonus/lightning/lnbits.md:
````diff
@@ -93,69 +93,121 @@ Table of contents
+  
+  ```ini
+  LND_REST_ENDPOINT=https://127.0.0.1:8080
   LND_REST_CERT=/home/lnbits/.lnd/tls.cert
````
```suggestion
  LND_REST_CERT="/home/lnbits/.lnd/tls.cert"
```

blckbx reviewed (commented):
LGTM, few nits and changes due to gh repo renaming

VajraOfIndra commented:
> LGTM, few nits and changes due to gh repo renaming

Thanks, all fixed, PTAL

blckbx reviewed (approved):
utACK 🔥
